CHANGELOG.md

```diff
@@ -2,7 +2,27 @@
 
 * Added method `vdc.QueryEdgeGateway` [#364](https://github.com/vmware/go-vcloud-director/pull/364)
 * Deprecated `vdc.GetEdgeGatewayRecordsType` [#364](https://github.com/vmware/go-vcloud-director/pull/364)
+* Added NSX-T Firewall Group (Security Groups and IP Set) support by using structures
```

Comment:
Suggesting below just because I got puzzled by the meaning of it all the way until I reached the very last file in the PR 😄 

```suggestion
* Added NSX-T Firewall Group type (which represents a Security Group or an IP Set) support by using structures
```

Reply:
Changed